api/src/bigquerytest/java/org/pmiops/workbench/cohorts/CohortMaterializationServiceBQTest.java

```diff
@@ -23,7 +23,6 @@
 import org.junit.runner.RunWith;
```

Comment:
Should probably have at least a minimal temporal test here to verify things are working end-to-end.